api/envoy/service/status/v3/csds.proto

```diff
@@ -124,10 +126,54 @@ message ClientConfig {
   option (udpa.annotations.versioning).previous_message_type =
       "envoy.service.status.v2.ClientConfig";
 
+  // GenericXdsConfig is a simpler and generic version of specifying
+  // XDS config status and the dump.
+  // [#next-free-field: 9]
+  message GenericXdsConfig {
+    // type_url represents the fully qualified name of xDS resource type
+    // like envoy.v3.Cluster, envoy.v3.ClusterLoadAssignment etc.
+    string type_url = 1;
+
+    // name of the xDS resource
+    string name = 2;
+
+    // This is the :ref:`version_info <envoy_api_field_service.discovery.v3.DiscoveryResponse.version_info>`
+    // in the last processed xDS discovery response. If there are only static bootstrap listeners, this
+    // field will be ""
```

Comment:
Line 133: Capitalize the first word?
Line 137: Capitalize the first word?
Line 140-142: Rewrap?

Reply:
Done.